build/textures.rs

```diff
@@ -644,11 +644,16 @@ fn build_texture<W: Write>(mut dest: &mut W, ty: TextureType, dimensions: Textur
 
             TextureDimensions::Texture2dArray => (write!(dest, "
                     let array_size = 0;
```

Comment:
Those were there before hand, and I've never seen the ``let {data, width, ...} = RawImage3d;``` syntax get used before, so thought it was needed.  Learned something new today at least :)